packages/@aws-cdk/aws-s3/test/notification-dests.ts

```diff
@@ -6,15 +6,15 @@ import cdk = require('@aws-cdk/cdk');
  * for AWS::SNS::Topic which implements IBucketNotificationDestination.
  */
 export class Topic extends cdk.Construct implements s3notifications.IBucketNotificationDestination {
-    public readonly topicArn: cdk.Arn;
+    public readonly topicArn: string;
     private readonly policy = new cdk.PolicyDocument();
     private readonly notifyingBucketPaths = new Set<string>();
 
     constructor(parent: cdk.Construct, id: string) {
         super(parent, id);
 
         const resource = new cdk.Resource(this, 'Resource', { type: 'AWS::SNS::Topic' });
-        const topicArn = new cdk.Ref(resource);
+        const topicArn = resource.ref;
```

Comment:
Wait, do we or do we not have `ref`?

Reply:
`cdk.Resource` has `getAtt` and `ref` as low level properties. cfn2ts generates properties that use them. This specific test case just uses `cdk.Resource`, so there are no generated accessors.